Index: nitzamon.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/nitzamon.py b/nitzamon.py
new file mode 100644
--- /dev/null	(date 1646843903753)
+++ b/nitzamon.py	(date 1646843903753)
@@ -0,0 +1,12 @@
+
+
+class Nitzamon:
+
+    def __init__(self, element, dmg, hp, spd, sprite, list_of_moves, name):
+        self.name = name
+        self.element = element
+        self.dmg = dmg
+        self.hp = hp
+        self.spd = spd
+        self.sprite = sprite
+        self.list_of_moves = list_of_moves
Index: move.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/move.py b/move.py
new file mode 100644
--- /dev/null	(date 1646844103483)
+++ b/move.py	(date 1646844103483)
@@ -0,0 +1,10 @@
+
+
+class Move:
+
+    def __init__(self, move_element, move_dmg, move_name):
+       self.move_element = move_element
+       self.move_dmg = move_dmg
+       self.move_name = move_name
+
+    def is_effective(self):
Index: quest.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/quest.py b/quest.py
new file mode 100644
--- /dev/null	(date 1646843511174)
+++ b/quest.py	(date 1646843511174)
@@ -0,0 +1,9 @@
+
+
+class Quest:
+
+    def __init__(self, is_completed, reward, dialogue):
+        self.is_completed = is_completed
+        self.reward = reward
+        self.dialogue = dialogue
+    
\ No newline at end of file
Index: dialogue.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/dialogue.py b/dialogue.py
new file mode 100644
--- /dev/null	(date 1646843732550)
+++ b/dialogue.py	(date 1646843732550)
@@ -0,0 +1,8 @@
+
+
+class Dialogue:
+
+    def __init__(self, text, sound):
+        self.text = text
+        self.sound = sound
+
