Index: Classes/Character Classes/NitzamonUser.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Character Classes/NitzamonUser.py b/Classes/Character Classes/NitzamonUser.py
new file mode 100644
--- /dev/null	(date 1646844436815)
+++ b/Classes/Character Classes/NitzamonUser.py	(date 1646844436815)
@@ -0,0 +1,6 @@
+import Character
+class NitzamonUser(Character):
+    def __init__(self, name, sprite, nitzamons, num_of_nitzamons):
+        super(self, name, sprite)
+        self.nitzamons = nitzamons
+        self.num_of_nitzamons = num_of_nitzamons
\ No newline at end of file
Index: Classes/Character Classes/NPC.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Character Classes/NPC.py b/Classes/Character Classes/NPC.py
new file mode 100644
--- /dev/null	(date 1646844399206)
+++ b/Classes/Character Classes/NPC.py	(date 1646844399206)
@@ -0,0 +1,6 @@
+import Character
+class NPC(Character):
+    def __init__(self, name, sprite, quests, small_talk):
+        super(self, name, sprite)
+        self.quests = quests
+        self.small_talk = small_talk
\ No newline at end of file
Index: Classes/Character Classes/Character.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\n\r\nclass Character:\r\n    pass
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Character Classes/Character.py b/Classes/Character Classes/Character.py
--- a/Classes/Character Classes/Character.py	(revision 54bcb7d31839889f95710dbc7ab664a103740654)
+++ b/Classes/Character Classes/Character.py	(date 1646843579422)
@@ -1,4 +1,4 @@
-
-
 class Character:
-    pass
\ No newline at end of file
+    def __init__(self, name ,pos):
+        self.name = name
+        self.pos = pos
